frontend/src/components/Displays/LocalizationVerification/ScheduleMissionChecklistDialog.tsx

```diff
@@ -0,0 +1,256 @@
+import { Button, Checkbox, Dialog, Typography } from '@equinor/eds-core-react'
+import { DeckMapView } from 'utils/DeckMapView'
+import { HorizontalContent, StyledDialog, VerticalContent } from './ScheduleMissionStyles'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import { Robot } from 'models/Robot'
+import { ChangeEvent, useState } from 'react'
+import { useInstallationContext } from 'components/Contexts/InstallationContext'
+
+interface ScheduleMissionChecklistDialogProps {
+    closeDialog: () => void
+    scheduleMissions: () => void
+    robot: Robot
+    missionDeckName: string
+}
+
+interface CheckDialogProps {
+    setIsCheckConfirmed: (b: boolean) => void
+    robot: Robot
+    deckName: string
+}
+
+const LocalisationDialog = ({ setIsCheckConfirmed, robot, deckName }: CheckDialogProps) => {
+    const { TranslateText } = useLanguageContext()
+    const { installationDecks } = useInstallationContext()
+
+    const newDeck = installationDecks.find((deck) => deck.deckName === deckName)
+
+    return (
+        <>
+            <Dialog.Header>
+                <Typography variant="h5">{TranslateText('Confirm placement of robot')}</Typography>
+            </Dialog.Header>
+            <Dialog.Content>
+                <VerticalContent>
+                    <Typography>
+                        {`${robot.name} (${robot.model.type}) ${TranslateText(
+                            'needs to be placed on marked position on'
+                        )} ${deckName} `}
+                        <b>{TranslateText('before')}</b>
+                        {` ${TranslateText('clicking confirm')}.`}
+                    </Typography>
+                    {newDeck && newDeck.defaultLocalizationPose && (
+                        <DeckMapView deck={newDeck} markedRobotPosition={newDeck.defaultLocalizationPose}></DeckMapView>
+                    )}
+                    <HorizontalContent>
+                        <Checkbox
+                            crossOrigin={undefined}
+                            onChange={(e: ChangeEvent<HTMLInputElement>) => setIsCheckConfirmed(e.target.checked)}
+                        />
+                        <Typography>
+                            {`${TranslateText('I confirm that')} ${robot.name} (${robot.model.type}) ${TranslateText(
+                                'has been placed on marked position on'
+                            )} `}
+                            <b>{deckName}</b>
+                        </Typography>
+                    </HorizontalContent>
+                </VerticalContent>
+            </Dialog.Content>
+        </>
+    )
+}
+
+const PressureDialog = ({ setIsCheckConfirmed, robot, deckName }: CheckDialogProps) => {
+    const { TranslateText } = useLanguageContext()
+
+    let statusText = ''
+    if (robot.pressureLevel) {
+        if (
+            robot.pressureLevel > robot.model.lowerPressureWarningThreshold &&
```

Comment:
![image](https://github.com/equinor/flotilla/assets/108514046/419249d3-ad1d-46cf-aa65-ed1cb3319ae8)

pressure is being compared in different unit of measurement